stories/demos/CommonFeatures.tsx

```diff
@@ -0,0 +1,170 @@
+import React, { useState, useCallback, useMemo } from 'react';
+import faker from 'faker';
+import { AutoSizer } from 'react-virtualized';
+import DataGrid, { SelectColumn, Column, RowsUpdateEvent, DEFINE_SORT } from '../../src';
+
+const dateFormatter = new Intl.DateTimeFormat(navigator.language);
+const currencyFormatter = new Intl.NumberFormat(navigator.language, {
+  style: 'currency',
+  currency: 'eur'
+});
+
+function TimestampFormatter({ timestamp }: { timestamp: number }) {
+  return <>{dateFormatter.format(timestamp)}</>;
+}
+
+function CurrencyFormatter({ value }: { value: number }) {
+  return <>{currencyFormatter.format(value)}</>;
+}
+
+interface Row {
+  id: number;
+  title: string;
+  assignee: string;
+  progress: number;
+  startTimestamp: number;
+  endTimestamp: number;
+  budget: number;
+}
+
+const columns: readonly Column<Row>[] = [
+  SelectColumn,
+  {
+    key: 'id',
+    name: 'ID',
+    width: 60,
+    frozen: true,
+    sortable: true
+  },
+  {
+    key: 'title',
+    name: 'Task',
+    width: 120,
+    editable: true,
+    frozen: true,
+    resizable: true,
+    sortable: true
+  },
+  {
+    key: 'assignee',
+    name: 'Assignee',
+    editable: true,
+    resizable: true,
+    sortable: true
+  },
+  {
+    key: 'progress',
+    name: 'Completion',
+    resizable: true,
+    sortable: true,
+    formatter(props) {
+      const value = props.row.progress;
+      return (
+        <>
+          <progress max={100} value={value} /> {Math.round(value)}%
+        </>
+      );
+    }
+  },
+  {
+    key: 'startTimestamp',
+    name: 'Start date',
+    resizable: true,
+    sortable: true,
+    formatter(props) {
+      return <TimestampFormatter timestamp={props.row.startTimestamp} />;
+    }
+  },
+  {
+    key: 'endTimestamp',
+    name: 'Deadline',
+    resizable: true,
+    sortable: true,
+    formatter(props) {
+      return <TimestampFormatter timestamp={props.row.endTimestamp} />;
+    }
+  },
+  {
+    key: 'budget',
+    name: 'Budget',
+    resizable: true,
+    sortable: true,
+    formatter(props) {
```

Comment:
`value = props.row[props.column.key]` is still an option, but that'd assume we always have a 1:1 mapping between column keys and row objects.

Reply:
@amanmahajan7 
I would agree with @nstepien , the reason why we removed the `value` was to not let the user think that's the exact 1:1 mapping. However, I am wondering if we should add an example formatter to display multiple fields at one time. Then in the example, it would be pretty intuitive so that the user would know they have such flexibility?

Reply:
I am okay with either approach